book/src/SUMMARY.md

```diff
@@ -57,6 +57,7 @@
   * [Snapshot Verification](proposals/snapshot-verification.md)
   * [Bankless Leader](proposals/bankless-leader.md)
   * [Slashing](proposals/slashing.md)
+  * [Leader Duplicate Block Slashing](proposals/leader-duplicate-block-slashing.md)
```

Comment:
Maybe leave "slashing" out of the name of this proposal, as it primarily deals with how we'll handle saving and reconciling multiple versions of a slot.

Ths represents a **_big_** change, since we settled on "one block per slot" last February.